stix_shifter/stix_translation/src/json_to_stix/json_to_stix_translator.py

```diff
@@ -194,11 +194,25 @@ def _transform(self, object_map, observation, ds_map, ds_key, obj):
 
                 DataSourceObjToStixObj._handle_cybox_key_def(key_to_add, observation, stix_value, object_map, object_name, group)
             else:
-                stix_value = DataSourceObjToStixObj._get_value(obj, ds_key, transformer)
-                if not DataSourceObjToStixObj._valid_stix_value(self.properties, key_to_add, stix_value):
-                    continue
+                # get the object name defined for custom attributes
+                if 'object' in ds_key_def:
+                    object_name = ds_key_def.get('object')
+                    # get the value mapped
+                    if 'value' in ds_key_def:
```

Comment:
So for my own understanding, if a value key is defined in the mapping, it's value will be written to the custom STIX attribute. 
```
"detail_resource_instanceDetails_networkInterfaces_0_publicIp": [
        {
          "key": "ipv4-addr.x_com_aws_ip_type",
          "object": "nc_public_ip",
          "value": "public",
          "cybox": false
        }
]
```
The above mapping example would give the following STIX object snippet:

```
"0": {
       "type": "ipv4-addr",
       "x_com_aws_ip_type": "public"
}
```

Reply:
Yes. Your understanding is correct

Reply:
Ok, I'll add it to the developer docs once this PR gets merged. I could see this being useful for other connectors as well when you want to hardcode a value into a STIX property.

Reply:
FYI, This value mapping code is added in else part of the code ie., when "cybox" = false is specified in mapping. It can't be used for standard STIX object.

Reply:
That's fine, though I can see a case where this may be useful for a standard STIX object as well.